modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -282,6 +282,9 @@ public static class Client {
 
         /** Resource not found. */
         public static final int RESOURCE_NOT_FOUND_ERR = CLIENT_ERR_GROUP.registerErrorCode((short) 10);
+
+        /** Client operation timeout. */
+        public static final int CLIENT_OPERATION_TIMEOUT_ERR = CLIENT_ERR_GROUP.registerErrorCode((short) 11);
```

Comment:
```suggestion
        public static final int OPERATION_TIMEOUT_ERR = CLIENT_ERR_GROUP.registerErrorCode((short) 11);
```

Please remove `CLIENT_` prefix to be consistent with other error names.